standard/classes.md

```diff
@@ -3099,27 +3100,39 @@ property_body
 property_initializer
     : '=' variable_initializer ';'
     ;
+
+ref_property_body
+    : '{' ref_get_accessor_declaration '}'
+    | '=>' 'ref' variable_reference ';'
+    ;
 ```
 
 *unsafe_modifier* ([§23.2](unsafe-code.md#232-unsafe-contexts)) is only available in unsafe code ([§23](unsafe-code.md#23-unsafe-code)).
 
+There are two kinds of *property_declaration*:
+
+- The first declares a non-ref-valued property. Its value has type *type*. This kind of property may be readable and/or writeable.
+- The second declares a ref-valued property. Its value is a *variable_reference* ([§9.4](variables.md#94-definite-assignment)), that may be `readonly`, to a variable of type *type*. This kind of property is only readable.
```

Comment:
Here be dragons!

I think this is consistent with other parts of the Standard and any change needs to ensure consistency. The definition as given is at *safe*, even if it could be less restrictive, so the issue can be punted till the next version.

But let’s see what others think…

Reply:
Marking as “resolved” here now #838 exists.